pootle/apps/pootle_language/forms.py

```diff
@@ -67,32 +71,40 @@ def search(self):
 
 class LanguageTeamAdminForm(LanguageTeamBaseAdminForm):
     rm_admins = forms.ModelMultipleChoiceField(
-        label=_("Admins"),
+        # Translators: This is members with rights to manage team.
+        label=_("Administrators"),
```

Comment:
We had a whole LONG LONG debate about this.  Please revert, that is what a comment is for